netlogo-gui/src/main/app/NotificationBanner.scala

```diff
@@ -0,0 +1,213 @@
+// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo
+
+package org.nlogo.app
+
+import com.vladsch.flexmark.html.HtmlRenderer
+import com.vladsch.flexmark.parser.Parser
+
+import java.awt.{BorderLayout, Dimension, Graphics}
+import java.awt.event.{ActionEvent, ActionListener, MouseAdapter, MouseEvent}
+import java.util.prefs.Preferences
+import javax.swing.{JButton, JEditorPane, JLabel, JOptionPane, JPanel, JScrollPane, SwingConstants}
+
+import org.nlogo.app.infotab.InfoFormatter
+import org.nlogo.core.I18N
+import org.nlogo.swing.{HoverDecoration, OptionDialog}
+import org.nlogo.theme.{InterfaceColors, ThemeSync}
+
+import org.json.simple.parser.JSONParser
+import org.json.simple.{JSONArray, JSONObject}
+
+import scala.io.Source
+
+
+case class JsonObject(eventId: Int, date: String, title: String, fullText: String)
+
+class NotificationBanner() extends JPanel with ThemeSync with HoverDecoration {
+  private var jsonObjectList: List[JsonObject] = List() // Initialize here first
```

Comment:
Most of these comments are just cluttering up the code without adding any new information that you wouldn't get from just reading the code.  I think that they should be removed.